Carbon/Functions/Assert-FirewallConfigurable.ps1

```diff
@@ -35,11 +35,15 @@ function Assert-FirewallConfigurable
 
     Use-CallerPreference -Cmdlet $PSCmdlet -Session $ExecutionContext.SessionState
 
-    if( (Get-Service 'Windows Firewall').Status -ne 'Running' ) 
+    if( (Get-Service -DisplayName 'Windows Firewall' -ErrorAction Ignore).Status -eq 'Running' )

```

Comment:
Take out the `-DisplayName` parameter so it is the same call as before. Carbon needs to  support all OSes that run PowerShell 4. This includes Windows 2008 R2. We don't have any 2008 R2 VMs to run our tests on, so this check has to stay *exactly* how it was, with the exception of the `-ErrorAction Ignore` parameter.

Reply:
Oh gotcha, that makes much more sense now. I'll change it back.